datasets/eu/esma_sanctions/eu_esma_sanctions.yaml

```diff
@@ -0,0 +1,25 @@
+title: "EU ESMA Sanctions"
+entry_point: crawler.py
+prefix: esma-sanctions
+coverage:
+  frequency: daily
+  start: 2024-03-12
```

Comment:
20 :)

Reply:
I think we need one in the yaml: `load_db_uri: ${OPENSANCTIONS_DATABASE_URI}`

Reply:
Fixed and added :)